terraform/gen.go

```diff
@@ -25,10 +25,11 @@ const (
 )
 
 var (
-	fOut    = flag.String("o", "main.tf", "output file")
-	fDot    = flag.String("dot", "../architecture.dot", "output DOT file")
-	fLegacy = flag.Bool("legacy", false, "use legacy functions in deployment")
-	fSeq    = flag.Bool("seq", false, "make the deployment sequential")
+	fOut      = flag.String("o", "main.tf", "output file")
+	fDot      = flag.String("dot", "../architecture.dot", "output DOT file")
+	fLegacy   = flag.Bool("legacy", false, "use legacy functions in deployment")
+	fSeq      = flag.Bool("seq", false, "make the deployment sequential")
+	fFetchers = flag.Bool("fetchers", false, "deploy node fetchers")
```

Comment:
```suggestion
	fFetchers = flag.Bool("fetchers", true, "deploy node fetcher CFs")
```
And then pass this flag to the codegen explicitly.

Reply:
Done.